adapter/internal/discovery/xds/server.go

```diff
@@ -383,6 +392,7 @@ func UpdateAPI(vHost string, apiProject mgw.ProjectAPI, environments []string) (
 		orgIDAPIMgwSwaggerMap[organizationID] = mgwSwaggerMap
 	}
 
+	environments := []string{deployedEnvironments[0].Name}
```

Comment:
Can apply a minor formatting improvement to this line.

Reply:
what is the formatting improvement we need here

Reply:
my bad I recognised it as a function closing curly brace.